patches/minecraft/net/minecraft/world/World.java.patch

```diff
@@ -535,40 +535,32 @@
                      }
                  }
              }
-@@ -2050,6 +2207,16 @@
+@@ -2050,7 +2207,7 @@
                          IBlockState iblockstate1 = this.func_180495_p(blockpos$pooledmutableblockpos);
                          Block block = iblockstate1.func_177230_c();
  
-+                        Boolean result = block.isEntityInsideMaterial(this, blockpos$pooledmutableblockpos, iblockstate1, p_72918_3_, (double)i3, p_72918_2_, false);
```

Comment:
This still needs to be called, to avoid breaking mods that are using it right now.

Reply:
I am calling it. That's why it has been added (back) in.

Reply:
That's not what I see with this patch.
This section with a call to `block.isEntityInsideMaterial` has been replaced by a call to `getCanPushEntity()`.

Reply:
Okay, you're right -- that is in World. I was looking at Entity where I had restored it.

Reply:
Okay, fixed. (Thanks for the careful review)